src/emulator/gui/src/gui.cpp

```diff
@@ -21,6 +21,9 @@
 
 #include <gui/imgui_impl_sdl_gl3.h>
 
+#ifdef NDEBUG
+#include <discord.h>
+#endif
```

Comment:
Think a comment here is necessary? I don't know how many people would be confused about this- but still.